docs/content/manual/pre-release/ha/backing-image-error-reporting-and-retry.md

```diff
@@ -10,12 +10,19 @@ title: Backing Image Error Reporting and Retry
 
 ## Backing image with sync failure
 1. Create a backing image. Then create and attach a volume using this backing image.
-2. Exec into the host of a worker node. remove the content of the backing image work directory then set the directory as immutable. This operation should fail one file of the backing image as well as keeping failing the following sync retries.
+2. Exec into the host of a worker node. remove the content of the backing image work directory then set the directory as immutable. 
+   This operation should fail one file of the backing image as well as keeping failing the following sync retries.
+   Option 1. Run the following command to remove the content of the backing image work directory and set the directory as immutable, which should cause the sync to fail.
     ```shell
-    rm -r /var/lib/longhorn/backing-images/<backing image name>-<backing image UUID>/
+    rm /var/lib/longhorn/backing-images/<backing image name>-<backing image UUID>/backing*
     chattr +i /var/lib/longhorn/backing-images/<backing image name>-<backing image UUID>/
     ```
-3. Monitor the backing image manager pod log. Verify the backoff works for the sync retry as well.
+   Option 2: Run the following command to remove the content of the backing image work directory and wait for the backing.tmp file to be generated, indicating that the sync is being retried. Once the retry succeeds, set the directory as immutable, which should cause the sync to fail
```

Comment:
I don't think we need to add the option2 

Before the fix of https://github.com/longhorn/longhorn/issues/5481
option1 would stuck the process and there would be no retry triggered

But after the fix , both option1 and option2 will keep triggering the retry until it succeed.

Reply:
Or we can still add it ~
but there is something need to fix in the description

Option 2: Run the following command to remove the content of the backing image work directory and wait for the backing.tmp file to be generated, **indicating that the sync has started and the file is transferring**. Set the directory as immutable during the transferring should still fail the sync **since it can't rename the tmp file to remove the `.tmp` extent in the end.**